src/storage/v3/delta.hpp

```diff
@@ -27,6 +29,11 @@ struct Edge;
 struct Delta;
 struct CommitInfo;
 
+inline uint64_t GetNextDeltaId() noexcept {
```

Comment:
```suggestion
inline uint64_t GetNextDeltaId() {
```

`WithLock` is not `noexcept`.